Add MediaVision explanation in overview.md of what-is-tizen (#1615)
Samsung/tizen-docs: docs/platform/what-is-tizen/overview.md

  ![IoTivity for Connectivity](media/about_tizen_6.png)

## AI and Machine Learning

> 

Reviewer: "MediaVision" should be written as a third heading to make it distinguishable from other third headings under the "AI and Machine Learning" section.
Ref: In other PRs (https://github.com/Samsung/tizen-docs/pull/1616 & https://github.com/Samsung/tizen-docs/pull/1611), different authors are working on different sections under the "AI and Machine Learning" section.
```suggestion
### MediaVision
```

> MediaVision  provides task based inference API; classification, detection, and pose landmark detection.

Reviewer: An extra space has been removed.
```suggestion
MediaVision provides task based inference API; classification, detection, and pose landmark detection.
```

It also provides easy ways for a user:

> - to run models with meta files which describe model information

Reviewer: The first word of a sentence should always be capitalized. Please look into this issue.
```suggestion
- To run models with metafiles which describe model information
```

> - to run various type of models with Machine Learning API

Reviewer: The first word of a sentence should always be capitalized. Please look into this issue.
```suggestion
- To run various types of models with Machine Learning API
```

> - to use H/W acceleration with NNRuntime

Reviewer: The first word of a sentence should always be capitalized. Please look into this issue.
```suggestion
- To use H/W acceleration with NNRuntime
```

   - Neural network model quantizer.
   - Various profiles and test scripts to evaluate performance on target.

### MediaVision
MediaVision provides task based inference API; classification, detection, and pose landmark detection.

> It also provides easy ways for a user:

Reviewer: When a colon introduces a list, the text that precedes the colon must be able to stand alone as a complete sentence.
Ref: https://developers.google.com/style/colons#introductory-phrase-preceding-colon
```suggestion
It also provides an easy way for a user to do the following:
```

> - To run models with metafiles which describe model information

Reviewer: Please remove the "to" from the beginning of the sentence.
```suggestion
- Run models with metafiles which describe model information
```

> - To run various types of models with Machine Learning API

Reviewer: Please remove the "to" from the beginning of the sentence.
```suggestion
- Run various types of models with Machine Learning API
```

> - To use H/W acceleration with NN Runtime

Reviewer: Please remove the "to" from the beginning of the sentence.
```suggestion
- Use H/W acceleration with NN Runtime
```
